pkg/catalog/types.go

```diff
@@ -143,9 +143,11 @@ type MeshCataloger interface {
 	// ListInboundTrafficTargetsWithRoutes returns a list traffic target objects composed of its routes for the given destination service account
 	ListInboundTrafficTargetsWithRoutes(service.K8sServiceAccount) ([]trafficpolicy.TrafficTargetWithRoutes, error)
 
-	// GetApexServicesForBackend returns a list of services that serve as the apex service in a traffic split where the
-	//	given service is a backend
-	GetApexServicesForBackend(service.MeshService) []service.MeshService
+	// IsApexService returns true if the given service is an apex service in any traffic split
+	IsApexService(service.MeshService) bool
+
+	// GetBackendServiceForApexService returns a list of backend services that serve the given apex service in a traffic split
+	GetBackendServiceForApexService(apexSvc service.MeshService) []service.MeshService
```

Comment:
This is internal to the `MeshCatalog` implementation, no reason to expose it in the interface.

Reply:
good catch! I have updated the code